src/main/java/uk/ac/ebi/eva/evaseqcol/entities/SeqColExtendedDataEntity.java

```diff
@@ -117,6 +118,20 @@ public static SeqColExtendedDataEntity<List<String>> constructSeqColNamesObjectB
         return seqColNamesObject;
     }
 
+    public static SeqColExtendedDataEntity<List<String>> constructSeqColNamesObjectByNamingConvention(
```

Comment:
Could we add a comment/docstring about the intended use for this method, or explicitly name it `...forTest` and hardcode the `TEST` value? Otherwise it's a bit confusing (and maybe even misleading) since it's actually always constructing the names list from the refseq value.

Reply:
updated